internal/app/init/internal/rootfs/rootfs.go

```diff
@@ -66,6 +66,12 @@ func Prepare(s string, inContainer bool, data *userdata.UserData) (err error) {
 		}
 	}
 
+	// Create required directories that are not part of FHS.
+	for _, p := range []string{"/etc/kubernetes/manifests", "/etc/cni", "/var/lib/kubelet", "/var/log/pods", "/usr/libexec/kubernetes"} {
```

Comment:
I found that we were creating the same well known paths in the `PreFunc` of multiple services. Since we were mistakenly running with a writable rootfs, it went unnoticed that these paths should be created _before_ switch rooting.